app/screens/gallery/gallery.js

```diff
@@ -34,10 +34,10 @@ export default class Gallery extends PureComponent {
 
         this.state = {
             index: props.index,
-            footerVisible: true,
+            headerVisible: true,
         };
 
-        this.footer = React.createRef();
+        this.header = React.createRef();
```

Comment:
this reference is being used in the footer component, I don't think you need to rename it